### DISCUSSION TITLE
4.6 release delivery date

### DISCUSSION THREAD

**Discussion by detienneZN** [JSQLParser/JSqlParser#1699] [General]
Hi,

What's the scheduled date for the 4.6 release delivery ?

Thanks

**Comment by manticore-projects** [comment]
Greetings!

Only @wumpz can decide. At the moment there are at least 3 major PRs pending approval and I would not bet on new release too soon. Although some Christmas Magic may happen, who knows.

In the meantime, you can always use the Snapshot Artifacts from Sonartype. From a practical point of view JSQLParser is pretty much a "rolling release" type of software and there is little meaning to the "big releases".

**Comment by wumpz** [comment]
The actual 4.6-SNAPSHOT is deployed at the moment. As @manticore-projects said there are some issues I'd like to have addressed before the next release is shipped.
